packages/vscode-extension/lib/wrapper.js

```diff
@@ -82,9 +82,12 @@ export function PreviewAppWrapper({ children, initialProps, ..._rest }) {
 
   const handleNavigationChange = useCallback(
     (navigationDescriptor) => {
+      const queryParams = new URLSearchParams(navigationDescriptor.params).toString();
       navigationHistory.set(navigationDescriptor.id, navigationDescriptor);
       devtoolsAgent?._bridge.send("RNIDE_navigationChanged", {
-        displayName: navigationDescriptor.name,
+        displayName: `${navigationDescriptor.name}${
```

Comment:
I'd much rather prefer for the name to already contain params for routes with parameters. We support expo router here only for the time being but the API is designed to be routing agnostic and for other solutions params may be formatted diferently.

The semantics of the "name" prop here is the "display name", so it is ok for the expo router plugin to format the params when appropriate

Reply:
@kmagiera got it sounds reasonable